API/src/routes/Assessment/index.js

```diff
@@ -9,18 +9,25 @@ module.exports = server => {
     `${ BASE_URL }/submit`,
     async (req, res, next) => {
       try {
-        const { assessment } = req.params;
+        //original
+        // const { assessment } = req.params;
+        const  assessment  = req.params;
     
         //verify that your data is making it here to the API by using console.log(assessment);
+        console.log('in api router')
+        console.log(assessment);
+        
         //call the AssessmentService.submit function from the API/src/microservices/Assessment/ and supply the correct parameters
-    
+        const ben  = await AssessmentService.submit(assessment);
+
         ResponseHandler(
           res,
-          `Submitted assessment`,
-          {},
+          '',
+          ben,
```

Comment:
Again, you aren't returning any data here, so I don't think you need to return the "ben" variable

Reply:
I changed this as recommended.